app/protocols/index.js

```diff
@@ -64,9 +66,10 @@ module.exports = {
   setupProtocols
 }
 
-function registerPriviledges () {
+function registerPriviledges() {
   globalProtocol.registerSchemesAsPrivileged([
     { scheme: 'hyper', privileges: P2P_PRIVILEDGES },
+    // { scheme: 'ssb', privileges: P2P_PRIVILEDGES }, // presumably errors because ssb uri are not url safe
```

Comment:
I'm not sure what priviledges make the most sense here, but I think the problem was with marking the SSB URI as being `standard` (e.g. on this line: https://github.com/AgregoreWeb/agregore-browser/blob/44df78d3096b80e8457b55ba3d6ff3d8e49d76d7/app/protocols/index.js#L4)

It might make sense to reuse the LOW_PRIVILEDGES value that's used for magnet links

Reply:
Thanks, low privileges is working :pray:

Reply:
I've done it like this to preserve the other privileges:

```js
{ scheme: 'ssb', privileges: { ...P2P_PRIVILEDGES, standard: false } }
```

Reply:
Maybe it'd be worth it to create a new priviledges object at the top for SSB? Not sure what to call it. Maybe P2P_URI_PRIVILEDGES?